core/assets/bundles/bundle_de.properties

```diff
@@ -251,33 +251,33 @@ map.random = [accent]Zufällige Karte
 map.nospawn = Diese Karte hat keine Kerne in denen die Spieler beginnen können! Füge einen [ROYAL]blue[] Kern zu dieser Karte im Editor hinzu.
 map.nospawn.pvp = Diese Karte hat keine gegnerischen Kerne wo Gegner starten könnten! Füge über den Editor [SCARLET] rote[] Kerne zu dieser Karte hinzu.
 map.nospawn.attack = Diese Karte hat keine gengnerischen Kerne, die Spieler angreifen können! Füge über den Editor [SCARLET] rote[] Kerne zu dieser Karte hinzu.
-map.invalid = Fehler beim Laden der Karte: Beschädigtes oder ungültige Karten Datei.
-workshop.update = Update Item
-workshop.error = Error fetching workshop details: {0}
-map.publish.confirm = Are you sure you want to publish this map?\n\n[lightgray]Make sure you agree to the Workshop EULA first, or your maps will not show up!
-workshop.menu = Select what you would like to do with this item.
-workshop.info = Item Info
+map.invalid = Fehler beim Laden der Karte: Beschädigte oder ungültige Karten-Datei.
+workshop.update = Objekt aktualisieren
+workshop.error = Fehler beim Holen der Werkstattdetails: {0}
+map.publish.confirm = Bist du sicher, dass du diesen Plan veröffentlichen willst?\n\n[lightgray]Stelle sicher, dass du der Werkstatt EULA zugestimmt hast, sonst wird der Plan nicht angezeigt!
+workshop.menu = Auswählen, was du mit diesem Objekt tun willst.
+workshop.info = Objektinformation
 changelog = Changelog (optional):
 eula = Steam EULA
-missing = This item has been deleted or moved.\n[lightgray]The workshop listing has now been automatically un-linked.
+missing = Dieses Objekt wurde gelöscht oder verschoben.\n[lightgray]Die Werkstattliste ist nun automatisch getrennt worden.
 publishing = [accent]Publishing...
 publish.confirm = Are you sure you want to publish this?\n\n[lightgray]Make sure you agree to the Workshop EULA first, or your items will not show up!
-publish.error = Error publishing item: {0}
+publish.error = Fehler beim Veröffentlichen des Objekts: {0}
 editor.brush = Pinsel
 editor.openin = Öffne im Editor
 editor.oregen = Erze generieren
 editor.oregen.info = Erze generiert:
-editor.mapinfo = Karten Info
+editor.mapinfo = Karteninformation
 editor.author = Autor:
 editor.description = Beschreibung:
-editor.nodescription = A map must have a description of at least 4 characters before being published.
+editor.nodescription = Ein Plan muss eine Beschreibung mit mindestens 4 Zeichen haben, damit er veröffentlicht werden kann.
```

Comment:
Eine Karte muss eine Beschreibung mit mindestens 4 Zeichen haben, damit er veröffentlicht werden kann.

Reply:
There's a mistake in your suggestion: **Eine** Karte ... damit **er** veröffentlicht

Reply:
Yeah forgot to correct that mistake also, its "sie".